CONTRIBUTING.md

```diff
@@ -0,0 +1,20 @@
+# How to contribute
+
+I'm really glad you're reading this, because we need volunteer developers to help this project.
+
+## Testing
+
+Our tests consists of RSpec examples. Please write RSpec examples for new code you create.
+
+## Submitting changes
+
```

Comment:
I think we can add some instruction about the fork
e.g. https://github.com/espoo-dev/espoo-dev/blob/main/CONTRIBUTING.md

Reply:
No sure, It may be good if it was mandatory, today is free to decide.

Reply:
I've added a section to contributors feel free to decide about it.